vertical-pod-autoscaler/README.md

```diff
@@ -45,13 +45,14 @@ procedure described below.
 
 # Installation
 
-The current default version is Vertical Pod Autoscaler 0.11.0
+The current default version is Vertical Pod Autoscaler 0.12.0
 
 ### Compatibility
 
 | VPA version     | Kubernetes version |
 |-----------------|--------------------|
-| 0.11            | 1.22+              |
+| 0.12            | 1.25+              |
+| 0.11            | 1.22 - 1.24        |
 | 0.10            | 1.22+              |
```

Comment:
Does 0.10 need to be changed in the same way as 0.11?

```suggestion
| 0.10            | 1.22 - 1.24        |
```

Reply:
I know that 0.10 won't work with 1.25

I'm not sure if it will work with 1.24. I think it should but before submitting a change like that I'd like to check (start a cluster, run full-vpa tests). Right now I don't have enough time to do the check